pom.xml

```diff
@@ -58,7 +58,7 @@
     </developers>
     <properties>
         <apache.rat.version>0.12</apache.rat.version>
-        <jacoco.maven.version>0.8.8</jacoco.maven.version>
+        <jacoco.maven.version>0.8.9-SNAPSHOT</jacoco.maven.version>
```

Comment:
Double-check: do we need this version to run against Java 20?

Reply:
@viniciusbrg ?

Can we stay with the 0.8.8 version?

Reply:
Sorry for the delay @otaviojava 

Using the 0.8.8 version causes some jacoco instrumentation exceptions during the tests, but it seems to affect only classes/libs provided by the JRE. The exceptions appear in the logs but the build doesn't fail.
It only failed when I used that java 20 profile, which caused all .class files to have major version = 64, which isn't supported by jacoco 0.8.8.

If these failures in the logs aren't a problem, I can revert to version 0.8.8 

Examples of exceptions:

```
java.lang.instrument.IllegalClassFormatException: Error while instrumenting sun/util/resources/cldr/provider/CLDRLocaleDataMetaInfo with JaCoCo 0.8.8.202204050719/5dcf34a.
	at org.jacoco.agent.rt.internal_b6258fc.CoverageTransformer.transform(CoverageTransformer.java:94)
	at java.instrument/java.lang.instrument.ClassFileTransformer.transform(ClassFileTransformer.java:244)
	at java.instrument/sun.instrument.TransformerManager.transform(TransformerManager.java:188)
	at java.instrument/sun.instrument.InstrumentationImpl.transform(InstrumentationImpl.java:541)
	at java.base/java.lang.ClassLoader.defineClass2(Native Method)
	at java.base/java.lang.ClassLoader.defineClass(ClassLoader.java:1104)
	at java.base/java.security.SecureClassLoader.defineClass(SecureClassLoader.java:182)
	at java.base/jdk.internal.loader.BuiltinClassLoader.defineClass(BuiltinClassLoader.java:821)
	at java.base/jdk.internal.loader.BuiltinClassLoader.findClassInModuleOrNull(BuiltinClassLoader.java:741)
	at java.base/jdk.internal.loader.BuiltinClassLoader.findClass(BuiltinClassLoader.java:621)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:633)
	at java.base/java.lang.Class.forName(Class.java:585)
	at java.base/java.lang.Class.forName(Class.java:560)
```

```
Caused by: java.lang.IllegalArgumentException: Unsupported class file major version 64
	at org.jacoco.agent.rt.internal_b6258fc.asm.ClassReader.<init>(ClassReader.java:199)
	at org.jacoco.agent.rt.internal_b6258fc.asm.ClassReader.<init>(ClassReader.java:180)
	at org.jacoco.agent.rt.internal_b6258fc.asm.ClassReader.<init>(ClassReader.java:166)
	at org.jacoco.agent.rt.internal_b6258fc.core.internal.instr.InstrSupport.classReaderFor(InstrSupport.java:280)
	at org.jacoco.agent.rt.internal_b6258fc.core.instr.Instrumenter.instrument(Instrumenter.java:77)
	at org.jacoco.agent.rt.internal_b6258fc.core.instr.Instrumenter.instrument(Instrumenter.java:109)
	... 98 more
```

Reply:
![image](https://user-images.githubusercontent.com/25652392/229562365-a1c30b33-4de9-4dca-a105-051db739fc42.png)

Here are the build results in my machine. The exceptions happen but it doesn't fail the build.

I'm running `mvn -B package --file pom.xml`

Reply:
I prefer more logs than using a SNAPSHOT version.

Reply:
Alright, I agree with you, using a snapshot version isn't ideal and can cause some problems too. Since these failures aren't blocking the build, I'll revert it, ok?

Reply:
Sure.

Reply:
Thank you.